templates/fb-instant-games/fbapp-config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "instant_games": {
+    "orientation": "PORTRAIT",
+    "override_web_orientation": "PORTRAIT",
+    "platform_version": "RICH_GAMEPLAY",
+    "navigation_menu_version": "NAV_FLOATING",
+    "custom_update_templates": {
+      "play_turn": {
+        "example": "Edgar played their move"
+      }
+    }
```

Comment:
what's this `custom_update_templates` field for?
can we remove it ?

Reply:
OK, we can remove it.